x-pack/plugins/infra/public/alerting/log_threshold/components/alert_details_app_section/index.tsx

```diff
@@ -4,87 +4,164 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
-import { ALERT_DURATION, ALERT_END } from '@kbn/rule-data-utils';
-import compact from 'lodash/compact';
+import React, { useEffect, useState } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { LIGHT_THEME } from '@elastic/charts';
+import { EuiPanel } from '@elastic/eui';
+import { ALERT_END, ALERT_EVALUATION_VALUE, ALERT_START } from '@kbn/rule-data-utils';
 import moment from 'moment';
-import React from 'react';
+import { useTheme } from '@emotion/react';
+import { EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import {
+  AlertAnnotation,
+  getPaddedAlertTimeRange,
+  AlertActiveTimeRangeAnnotation,
+} from '@kbn/observability-alert-details';
+import { useEuiTheme } from '@elastic/eui';
+import { UI_SETTINGS } from '@kbn/data-plugin/public';
+import { useKibanaContextForPlugin } from '../../../../hooks/use_kibana';
 import { getChartGroupNames } from '../../../../../common/utils/get_chart_group_names';
-import { type PartialCriterion } from '../../../../../common/alerting/logs/log_threshold';
+import {
+  ComparatorToi18nMap,
+  ComparatorToi18nSymbolsMap,
+  type PartialCriterion,
+} from '../../../../../common/alerting/logs/log_threshold';
 import { CriterionPreview } from '../expression_editor/criterion_preview_chart';
-import { AlertAnnotation } from './components/alert_annotation';
 import { AlertDetailsAppSectionProps } from './types';
+import { Threshold } from '../../../common/components/threshold';
 
 const LogsHistoryChart = React.lazy(() => import('./components/logs_history_chart'));
 
-const AlertDetailsAppSection = ({ rule, alert }: AlertDetailsAppSectionProps) => {
-  const ruleWindowSizeMS = moment
-    .duration(rule.params.timeSize, rule.params.timeUnit)
-    .asMilliseconds();
-  const alertDurationMS = alert.fields[ALERT_DURATION]! / 1000;
-  const TWENTY_TIMES_RULE_WINDOW_MS = 20 * ruleWindowSizeMS;
+const AlertDetailsAppSection = ({
+  rule,
+  alert,
+  setAlertSummaryFields,
+}: AlertDetailsAppSectionProps) => {
+  const [selectedSeries, setSelectedSeries] = useState<string>('');
+  const { uiSettings } = useKibanaContextForPlugin().services;
+  const { euiTheme } = useEuiTheme();
+  const theme = useTheme();
+  const timeRange = getPaddedAlertTimeRange(alert.fields[ALERT_START]!, alert.fields[ALERT_END]);
+  const formatValue = (threshold: number) => String(threshold);
```

Comment:
**nit:** being a pure function, you can move it outside the component declaration.